client/my-sites/marketing/buttons/buttons.jsx

```diff
@@ -89,7 +85,10 @@ class SharingButtons extends Component {
 
 	UNSAFE_componentWillReceiveProps( nextProps ) {
```

Comment:
This could be very straightforwardly converted to `componentDidUpdate`?

Reply:
Addressed in https://github.com/Automattic/wp-calypso/pull/56579/commits/29a0fa1f8cbb06453d373db782b8a02e30f4d57b